crates/rattler_index/src/main.rs

```diff
@@ -32,13 +32,18 @@ struct Cli {
 
     /// The maximum number of packages to process in-memory simultaneously.
     /// This is necessary to limit memory usage when indexing large channels.
-    #[arg(long, default_value = "128", global = true)]
+    #[arg(long, default_value = "32", global = true)]
```

Comment:
when testing it out with 128 on aws s3, i regularly got errors that weren't there when limiting ourselves to 32 maximum concurrent downloads